lib/Validation/ValidationMessage.js

```diff
@@ -0,0 +1,36 @@
+import styled, { css } from 'styled-components';
+import ErrorIcon from 'lib/assets/icon-error.svg';
+import SuccessIcon from 'lib/assets/icon-success.svg';
+
+export const ValidationMessage = styled.p`
+  margin: 0;
+  font-size: 0.75rem;
+  line-height: 1.05rem;
+  min-height: 1.1875rem;
+  padding-left: 1.923125rem;
+  background-position: left 0.125rem;
+  background-repeat: no-repeat;
+  background-image: url(${({ error }) => (error ? ErrorIcon : SuccessIcon)});
+  color: ${({ theme, error }) => (error ? theme.palette.error : theme.palette.generics.black)};
+  em {
+    font-size: 0.75rem;
+    color: ${({ theme }) => theme.palette.secondary};
+    line-height: 0;
+  }
+  ${({ darkContextOnMobile }) =>
```

Comment:
This is kinda ugly. I would prefer having a `dark` prop and rendering the component twice, adding an extra class to show/hide in desktop/mobile.
I can live with it tho.

Reply:
"ugly"? It's HORRIBLE, but I wanted to avoid rendering the same twice, but if that's ok, I'll do that.